.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+### Contribution Best Practices
+
+**Feature Requests and Bug Reports**
+
+* Some newcomers are not accustomed to issue trackers and might try to inform developers on the mailing list, 
+chat or even write private emails about bugs and features, but the right place to do this is: The issue tracker of a project.
+
+* Any bug or feature, please open an issue in the issue tracker right away and indicate if you want to work on it yourself.
+
+* When you file a bug report on the issue tracker, make sure you add steps to reproduce it. Especially if that bug is some weird/rare one.
+
+* In case you are not sure if the issue is an actual issue or not, please add the question label to it.
+
+**Working on issues**
```

Comment:
This section is not useful. Most of the things are already defined in Github Workflow.

Reply:
Text should be kept short and practical. No bureaucratic idioms.